Apps/Examples/Examples/SiwftUI Examples/AnnotationsExample.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct AnnotationsExample: View {
+    struct City {
+        var name: String
+        var location: CLLocationCoordinate2D
+    }
+
+    @State var camera = CameraState(center: .zero, zoom: 1)
+
+    var cities = [
+        City(name: "Helsinki", location: .helsinki),
+        City(name: "New York", location: .newYork),
+        City(name: "London", location: .london),
+        City(name: "Berlin", location: .berlin)
+    ]
+
+    var body: some View {
+        MapboxView(camera: $camera)
+            .annotations(cities.map {
```

Comment:
I guess this won't allow us to change annotations in runtime?
I would expect this method to accept a `Binding` but we need to be cautious here not to recreate the whole MapView each time annotations change.